app/controllers/api/doughnuts_controller.rb

```diff
@@ -1,50 +1,47 @@
 class Api::DoughnutsController < ApplicationController
   def index
```

Comment:
Add the following here so that an unauthenticated user may access the list of every donut:
```
skip_before_action :authorized
```

Reply:
If we are using other doughnut actions, Add this line instead:
`skip_before_action :authorized, only: [:index]`
This will still require authentication for adding or modifying donuts.